examples/cpu_render.rs

```diff
@@ -68,12 +34,11 @@ fn main() {
                 {
                     tree.insert(
                         &V3c::new(x, y, z),
-                        RGB::new(
-                            (255 as f32 * x as f32 / tree_size as f32) as u8,
-                            (255 as f32 * y as f32 / tree_size as f32) as u8,
-                            (255 as f32 * z as f32 / tree_size as f32) as u8,
-                            255,
-                        ),
+                        Albedo::default()
+                            .with_red((255 as f32 * x as f32 / tree_size as f32) as u8)
+                            .with_green((255 as f32 * y as f32 / tree_size as f32) as u8)
+                            .with_blue((255 as f32 * z as f32 / tree_size as f32) as u8)
+                            .with_alpha(255),
```

Comment:
This is just :trophy: 

In the future the scale/format of the RGB values could be abstracted away, so there's also support for 0..1 ranges